tests/main/security-device-cgroups-serial-port/task.yaml

```diff
@@ -16,58 +16,59 @@ restore: |
         rm -f /dev/ttyS4 /dev/ttyS4.spread
     fi
 
+    systemctl restart systemd-udevd
     udevadm control --reload-rules
     udevadm trigger
 
 execute: |
-    if [ "$SPREAD_REBOOT" = 0 ]; then
-        echo "Given a snap is installed"
-        "$TESTSTOOLS"/snaps-state install-local test-snapd-sh
+    echo "Given a snap is installed"
```

Comment:
I see that you didn't originally write this, but maybe it's a good occasion to fix this comment (I guess that it should be "given snap is installed").